app/views/layouts/exercise_inputs/layouts/_input_kids.html.erb

```diff
@@ -20,12 +20,13 @@
 
         <div class="mu-kids-states">
           <div class="mu-kids-state mu-state-initial mu-kids-gbs-board-initial">
-            <strong>Tablero inicial</strong>
+            <strong id="initialBoardText"><%= t :initial_board %></strong>
```

Comment:
Also, Ids should be `mu-initial-state-text` and `mu-actual-state-text`

Reply:
I can change it in both projects!